spec/models/assign_sp_component_to_service_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe AssignSpComponentToServiceEvent, type: :model do
+
+  let(:service) { create(:service) }
+  let(:component1) { create(:sp_component) }
```

Comment:
a bit pedantic... but it would be more rubyish to use an underscore like so: `:component_1`

Reply:
Please do be pedantic, I need to learn!  Surprised the linter allowed this.